src/Config/MosaicConfig.ts

```diff
@@ -126,12 +131,26 @@ export default class MosaicConfig {
       const config = fs.readFileSync(filePath).toString();
       if (config && config.length > 0) {
         const jsonObject = JSON.parse(config);
+        MosaicConfig.validateSchema(jsonObject);
         return new MosaicConfig(jsonObject);
       }
     }
     return new MosaicConfig({} as any);
   }
 
+  /**
+   * This method validate json object against mosaic config schema. This methods throws a custom exception if validation fails.
+   * @param jsonObject JSON object to be validated against schema.
+   */
+  private static validateSchema(jsonObject) {
```

Comment:
What is the type for jsonObject?

Reply:
Till it's not validated, it `any` kind. I have added the input type and return.